cardano-node/src/Cardano/Common/Parsers.hs

```diff
@@ -46,19 +47,31 @@ parseNodeId desc =
 parseProtocol :: Parser Protocol
 parseProtocol = asum
   [ fl ByronLegacy "byron-legacy"
-    "Use the Byron/Ouroboros Classic suite of algorithms"
+    "Byron/Ouroboros Classic suite of algorithms"
 
   , fl BFT "bft"
-    "Use the BFT consensus algorithm"
+    "BFT consensus"
 
   , fl Praos "praos"
-    "Use the Praos consensus algorithm"
+    "Praos consensus"
 
   , fl MockPBFT "mock-pbft"
-    "Use the Permissive BFT consensus algorithm using a mock ledger"
+    "Permissive BFT consensus with a mock ledger"
 
   , fl RealPBFT "real-pbft"
-    "Use the Permissive BFT consensus algorithm using the real ledger"
+    "Permissive BFT consensus with a real ledger"
+  ]
+  where
+    fl :: forall a. a -> String -> String -> Parser a
```

Comment:
`fl` is duplicated below. Factoring out with a comment would be nice :).

Reply:
@deepfire: What does `fl` stand for?

Reply:
Done.